### ISSUE TITLE
Investigate club functionality

### ISSUE THREAD

**Issue by tarolling** [tarolling/rps-bot-old#8]
Error logs for `join`
```shell
TypeError: Cannot read properties of null (reading 'value')
    at module.exports (/home/minecraft/multicraft/servers/server776960/src/clubs/db/join.js:30:25)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Object.execute (/home/minecraft/multicraft/servers/server776960/commands/rps/club.js:75:39)
    at async Object.execute (/home/minecraft/multicraft/servers/server776960/events/interactionCreate.js:13:13)
```

**Comment by tarolling** [comment]
Error logs for `leave`
```shell
TypeError: Cannot read properties of undefined (reading 'name')
    at module.exports (/home/minecraft/multicraft/servers/server776960/src/clubs/db/leave.js:27:83)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Object.execute (/home/minecraft/multicraft/servers/server776960/commands/rps/club.js:82:39)
    at async Object.execute (/home/minecraft/multicraft/servers/server776960/events/interactionCreate.js:13:13)
```

**Comment by tarolling** [comment]
More error logs
```shell
TypeError: Cannot read properties of null (reading 'available')
    at Object.execute (/home/minecraft/multicraft/servers/server776960/commands/rps/club.js:66:20)
    at Object.execute (/home/minecraft/multicraft/servers/server776960/events/interactionCreate.js:13:27)
    at Client.<anonymous> (/home/minecraft/multicraft/servers/server776960/bot.js:48:50)
    at Client.emit (node:events:519:28)
    at InteractionCreateAction.handle (/home/minecraft/multicraft/servers/server776960/node_modules/discord.js/src/client/actions/InteractionCreate.js:97:12)
    at module.exports [as INTERACTION_CREATE] (/home/minecraft/multicraft/servers/server776960/node_modules/discord.js/src/client/websocket/handlers/INTERACTION_CREATE.js:4:36)
    at WebSocketManager.handlePacket (/home/minecraft/multicraft/servers/server776960/node_modules/discord.js/src/client/websocket/WebSocketManager.js:355:31)
    at WebSocketManager.<anonymous> (/home/minecraft/multicraft/servers/server776960/node_modules/discord.js/src/client/websocket/WebSocketManager.js:239:12)
    at WebSocketManager.emit (/home/minecraft/multicraft/servers/server776960/node_modules/@vladfrangu/async_event_emitter/dist/index.cjs:285:31)
    at WebSocketShard.<anonymous> (/home/minecraft/multicraft/servers/server776960/node_modules/@discordjs/ws/dist/index.js:1190:51)
```

**Comment by tarolling** [comment]
resolved by a59cc8046ba79d41cfc7f7aa8a2a545eca0aa2a9
